app/summarizer/head.tsx

```
@@ -0,0 +1,116 @@
+            ],
+            url,
+            screenshot: ogImage,
+          }),
+        }}
+      />
+
+      {/* Research-specific Structured Data */}
+      <script
+        type="application/ld+json"
+        // eslint-disable-next-line react/no-danger
+        dangerouslySetInnerHTML={{
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)